File: CloudMusic/src/main/java/com/ken/android/CloudMusic/Interface/MediaPlayerController.java
package com.ken.android.CloudMusic.Interface;

import com.ken.android.CloudMusic.FilesRead.MusicInfo;

/**
 * Created by axnshy on 16/8/9.
 */
public interface MediaPlayerController {
    public abstract void play(MusicInfo musicInfo);
    public abstract void pause();
    public abstract void previous();
    public abstract void next();
    public abstract void repeat();
    public abstract void shuffle();
}

File: SmartPlay/src/main/java/com/luffy/smartplay/Interface/MediaPlayerController.kt
package com.luffy.smartplay.Interface

import com.luffy.smartplay.db.bean.MusicData

/**
 * Created by axnshy on 16/8/9.
 */
interface MediaPlayerController {
    fun play(musicInfo: MusicData?)
    fun pause()
    fun previous()
    operator fun next()
    fun repeat()
    fun shuffle()
}